src/apps/rate_limiter/rate_limiter.lua

```diff
@@ -25,12 +25,15 @@ local PACKET_SIZE = 60
 
 function RateLimiter:new (arg)
    local conf = arg and config.parse_app_arg(arg) or {}
-   assert(conf.rate)
-   assert(conf.bucket_capacity)
+   --- By default, limit to 10 Mbps, just to have a default.
+   conf.rate = conf.rate or (10e6 / 8)
+   -- By default, allow for 255 standard packets in the queue.
+   conf.bucket_capacity = conf.bucket_capacity or (255 * 1500)
    conf.initial_capacity = conf.initial_capacity or conf.bucket_capacity
    local o =
    {
       rate = conf.rate,
+      leaky = conf.leaky,
```

Comment:
Incidentally, boolean configuration values don't default well to "true"; you end up having to interpret a `nil` value to mean the default, when for the user it could be simply that they meant to specify a `nil` value as the boolean value for the parameter (which is a perfectly valid false value of course).  So if we control this via a boolean parameter I think it should be phrased in a way that the default is false :)